django/applications/catmaid/control/cropping.py

```diff
@@ -136,14 +179,14 @@ def get_image(self):
         except requests.exceptions.RequestException as e:
             raise ImageRetrievalError(self.path, str(e))
 
-        blob = Blob( img_data )
-        image = Image( blob )
-        # Check if the whole image should be used and cropped if necessary.
-        src_width = image.size().width()
-        src_height = image.size().height()
+        image = PILImage.open(BytesIO(img_data))
+
+        src_width, src_height = image.size
+
         if self.width != src_width or self.height != src_height:
-            box = Geometry( self.width, self.height, self.x_min_src, self.y_min_src )
-            image.crop( box )
+            # left upper right lower
+            image = image.crop((self.x_min_src, self.y_min_src, self.x_min_src + self.width, self.y_min_src + self.height))
+            image.load()
```

Comment:
This `load` may or may not be necessary (`crop` is implemented in a lazy fashion and is *probably* realised whenever the image data is actually required), but it seemed more intuitive to include it.

Reply:
You are right it doesn't seem to be needed. I tested it without and it works too. Not knowing much about `PIL` I feel removing it is clearer (at least I wasn't expecting the `load()` call there).

Reply:
Since I merged this branch already, I applied this change and removed `load()`.